python_modules/dagster-graphql/dagster_graphql/implementation/fetch_assets.py

```diff
@@ -566,7 +570,8 @@ def build_partition_statuses(
         graphene_ranges = []
         for r in ranges:
             partition_key_range = cast(
-                TimeWindowPartitionsDefinition, materialized_partitions_subset.partitions_def
+                TimeWindowPartitionsDefinition,
+                materialized_partitions_subset.partitions_def,
```

Comment:
i think ruff ran from a different working directory and added all these commas, i can try to undo it if anyone cares